### ISSUE TITLE
JVM and Android artifacts?

### ISSUE THREAD

**Issue by ragunathjawahar** [spotify/diffuser#9]
Folks, I wasn't able to find the diffuser artifact on Maven Central or JCenter. Is this on the horizon anytime soon?

**Comment by togi** [comment]
Yes, absolutely, it's pretty much the first thing on the list of things to do for diffuser. Most likely it'll be published on maven central. I'll be updating the documentation with instructions once the artifacts are available.
